### DISCUSSION TITLE
How to disable logs?

### DISCUSSION THREAD

**Discussion by jewelshkjony** [iBotPeaches/Apktool#3629] [Q&A] [answered]
Actually I'm working on a CLI project and here I don't want to show any logs for decompiling/recompiling the apk.
If its possible with the current version 2.9.3 so please let me know. Otherwise you might add a command to disable logs.
(Thanks for the awesome & powerful tool)

**Comment by iBotPeaches** [comment, marked as answer]
`-q` - will have 0 output.

This is explained in docs here: https://apktool.org/docs/cli-parameters#common-options
